### PR TITLE
Remove leftover v4 forms

### PR DESCRIPTION
**Pull request by ysds**

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -636,6 +635,7 @@ $form-range-thumb-focus-box-shadow:        0 0 0 1px $body-bg, $input-focus-box-
 $form-range-thumb-focus-box-shadow-width:  $input-focus-width !default; // For focus box shadow issue in IE/Edge
 $form-range-thumb-active-bg:               lighten($component-active-bg, 35%) !default;
 $form-range-thumb-disabled-bg:             $gray-500 !default;
+$form-range-thumb-transition:               background-color .15s ease-in-out, border-color .15s ease-in-out, box-shadow .15s ease-in-out !default;
```
Comment (mdo) [inline scss/_variables.scss]: Should we make this available to all other forms like checks/radios?

**Review by mdo — COMMENTED** [review]
Looks good overall, just one question.

File: scss/_variables.scss (hunk 1 above)
Comment (ysds) [reply scss/_variables.scss]: Yep! I was aware of that when I saw the TODO comment in the `_form-check.scss`, but I prioritize removing "custom" in this PR.

**Review by mdo — APPROVED** [review]
Reminds me I need to review the form layout options alongside the grid ideas from @MartijnCuppens.

**Review by MartijnCuppens — APPROVED** [review]
Yeah, gonna update the grid PR one of these days since the forms update in now merged. Anyway, this LGTM.

**Pull request merged**